src/core/nixl_listener.cpp

```diff
@@ -663,6 +663,58 @@ void nixlAgentData::getCommWork(std::vector<nixl_comm_req_t> &req_list){
     commQueue.clear();
 }
 
+nixl_status_t
+nixlAgentData::loadConnInfo(const std::string &remote_name,
+                            const nixl_backend_t &backend,
+                            const nixl_blob_t &conn_info) {
+    if (backendEngines.count(backend) == 0) {
+        NIXL_ERROR << "Agent " << name << " does not support a remote backend: " << backend;
```

Comment:
Are you sure we want to print an error here? Previously, this flow did not print an error. Maybe NIXL_DEBUG?